newsfragments/2611.bugfix.rst

```diff
@@ -1 +1,4 @@
-With ``strict_exception_groups=True``, when you ran a function in a nursery which raised an exception before calling ``task_status.started()``, it previously got wrapped twice over in ``ExceptionGroup`` in some cases. It no longer does that, and also won't wrap any ``ExceptionGroup`` raised by the function itself.
+When a starting function raises before calling :func:`trio.TaskStatus.started`,
+:func:`trio.Nursery.start` will no longer wrap the exception in an undocumented
+:exc:`ExceptionGroup`. Previously, an :exc:`ExceptionGroup` was added when using
+``trio.run(..., strict_exception_groups=True)``.
```

Comment:
I think it's also good to mention in particular that this led to double-wrapped exceptions if used together with `trio.Nursery.start` with `strict_exception_groups` set to `True` or `None`.